### ISSUE TITLE
Are necessary any implicit changes made by GitLocalize?

### ISSUE THREAD

**Issue by buslov** [k88hudson/git-flight-rules#204]
GitLocalize adds some changes when making a pull-request:
* Replaces * by - in lists
![image](https://user-images.githubusercontent.com/33098513/34785786-35aeb548-f643-11e7-9613-90327ebd7317.png)
* Extra empty lines
![image](https://user-images.githubusercontent.com/33098513/34787850-c9c68b38-f649-11e7-9f6d-de746ffebcf0.png)
* Completely breakes TOC's links
![image](https://user-images.githubusercontent.com/33098513/34786235-9c6797ea-f644-11e7-991e-246dbf35ec8b.png)
* Deletes redundant backquotes
![image](https://user-images.githubusercontent.com/33098513/34788395-7f582a6e-f64b-11e7-8d41-9bcfaff9a2c8.png)
* Changes formatting characters:
![image](https://user-images.githubusercontent.com/33098513/34788434-9f8e7bbc-f64b-11e7-9a7f-e01f2ae1e487.png)
* Decodes (some?) HTML entities:
![image](https://user-images.githubusercontent.com/33098513/34788449-a9f6d342-f64b-11e7-9286-00f6f8a043a0.png)

**Comment by RichardLitt** [comment]
@sotayamashita these changes seem out of scope for a translation engine. Are they necessary?

**Comment by sotayamashita** [comment]
@RichardLitt  GitLocalize change some markdown following flow. We parse markdown and convert it into HTML to translate. After translating, we convert HTML to markdown based on [commonmark spec](http://commonmark.org/). we understand that issue but it will rendered same before. 

What do you think?

**Comment by RichardLitt** [comment]
Good to know. You're right, most of these are superficial. However, the TOC changes aren't - as far as I know, they will break the links. @buslov can you confirm this?

**Comment by buslov** [comment]
@RichardLitt, you are right. I suppose that GitLocalize just copies links from English source. We could add some tests (#205) to Travis CI to prevent any merges with broken links.
